packages/sessions-sdk-react/src/components/token-list.tsx

```diff
@@ -100,9 +104,13 @@ export const TokenList = ({
                   </div>
                 </div>
                 <div className={styles.amountAndActions}>
-                  <div className={styles.amountAndSymbol}>
-                    <span className={styles.amount}>{amountAsString}</span>
-                    {symbol && <span className={styles.symbol}>{symbol}</span>}
+                  <div className={styles.amountAndDetails}>
+                    <span className={styles.amount}>{amountAsString}{symbol && ` ${symbol}`}</span>
+                    {notionalValue !== undefined && (
+                      <span className={styles.notional}>
+                        ${notionalValue.toLocaleString(undefined, { minimumFractionDigits: 2, maximumFractionDigits: 2 })}
```

Comment:
As I'd mentioned above it would be better to use `dnum` to format so we can stick to using bigint calculations